buildbot-conf/cordova.conf

```diff
@@ -229,6 +229,8 @@ plugins_cleanup_steps = [
     NPMInstall(command=['shelljs'], what='shelljs'),
     RMRF('~/.cordova/*', description='removing cache', haltOnFailure=False),
     RMRF(BASE_WORKDIR + '/*', description='cleaning workspace', haltOnFailure=False),
+    RMRF(BASE_WORKDIR + '/*', description='cleaning workspace again', haltOnFailure=False),
+    RMRF(BASE_WORKDIR + '/*', description='cleaning workspace yet again', haltOnFailure=True),
```

Comment:
The solid solution, and hence the "proper fix", is to offload this work to a custom cleanup script. We currently have a chicken-and-egg problem because we need to download the script before we clean up, and then clean it up using itself. In the near future we should consolidate the medic scripts into an executable called `medic` that is not part of the downloaded repositories, and we will be able to clean up by calling something like:

    medic clean
    medic update_config
    medic build windows

This will move a lot of logic out of buildbot, and it will also make the automated build tools more usable outside of buildbot.

For now, the three steps are a very simple effective fix for the intermittent problem.